### ISSUE TITLE
Feature Request: AES Encryption for User Data

### ISSUE THREAD

**Issue by Tijimonster** [clipto-pro/Desktop#12]
I totally love the way Clipto works, but I copy sensitive data many, many times on my phone and my computer that I would never want out in the open for anyone to see. Could you add encryption to the transfer process so not just anyone on my Wi-fi network can see exactly what I'm copying?

Thanks!

**Comment by atrashler** [comment]
Yes. AES encryption will be added also. We have such plans. It requires some more time to implement it correctly and simply to use.

**Comment by atrashler** [comment]
> I totally love the way Clipto works, but I copy sensitive data many, many times on my phone and my computer that I would never want out in the open for anyone to see. Could you add encryption to the transfer process so not just anyone on my Wi-fi network can see exactly what I'm copying?
> 
> Thanks!

Yes. AES encryption will be added also. We have such plans. It requires some more time to implement it correctly and simply to use.

**Comment by Fohroer** [comment]
Hello, dear developer!

In which year and month could we expect the implementation of this functionality, according to the roadmap?

**Comment by atrashler** [comment]
> Hello, dear developer!
> 
> In which year and month could we expect the implementation of this functionality, according to the roadmap?

Hi!

Now I want to:

1) complete with the general functionality first on Android, Desktop and Web
2) implement iOS version
3) add end-to-end encryption

Well... It is difficult to predict, because, based on my current experience, what I have planned does not happen in time due to limited resources (I am working on the product in spare time, because it is not profitable). Let things take their course. Sooner or later it will be realized. It's not just up to me, it's up to the community:)
